setup.cfg

```diff
@@ -1,16 +1,16 @@
 [metadata]
 name = EpikCord.py
-version = attr: EpikCord.__version__
+version = attr: EpikCord.__init__.__version__
```

Comment:
Is this needed? Surely because __version__ is in the __init__.py, it is exported by default and this isn't needed?

Reply:
No it's not just a test x)
Let me revert this